apps/rpe/sbox/base/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../../../base
+  - ../../base
```

Comment:
Might be intentional ?

Reply:
Do you know potential reason? RPE namespace isn't created managed identity - federated credentials etc

Reply:
Just add workload identity like preview i think. Usually we dont add everything in base to preview and sbox to avoid deploying unnecessary apps.

Reply:
Ah of course - done